addons/binding/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/model/SomfyTahomaGetEventsResponse.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2010-2018 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.somfytahoma.model;
+
+import java.util.ArrayList;
+
+/**
+ * The {@link SomfyTahomaGetEventsResponse} holds information about response to getEvents
+ * command of your TahomaLink account.
+ *
+ * @author Ondrej Pecta - Initial contribution
+ */
+public class SomfyTahomaGetEventsResponse {
+
+    ArrayList<SomfyTahomaEvent> events;
```

Comment:
`Collection<SomfyTahomaEvent> events = Collections.emptyList();`

otherwise you might run into NPEs because your `getEvents()` method return `null`.

Reply:
OK